zetaclient/bitcoin_client.go

```diff
@@ -825,8 +898,8 @@ func (ob *BitcoinChainClient) loadDB(dbpath string) error {
 		return err
 	}
 
-	//Load submitted transactions
-	err = ob.BuildSubmittedTxMap()
+	//Load included transactions
+	err = ob.BuildIncludedTxMap()
 	if err != nil {
```

Comment:
```suggestion
	if err = ob.BuildIncludedTxMap(); err != nil {
```
Just a suggestion, I personally find this format more visual